src/outbox/outboxReducers.js

```diff
@@ -9,6 +9,7 @@ const reducer = (state: OutboxState = initialState, action: Action): OutboxState
     case MESSAGE_SEND: {
       return [...state, { ...action.params }];
     }
+    case DELETE_OUTBOX_MESSAGE:
```

Comment:
wondering whats the use case here? Is it intended to change the state like the `EVENT_NEW_MESSAGE` case?
Also might be useful to add a test for this?